crates/examples/examples/simple_network_example.rs

```diff
@@ -2,19 +2,13 @@
 
 use configuration::NetworkConfig;
 use orchestrator::Orchestrator;
-use provider::NativeProvider;
-use support::{fs::local::LocalFileSystem, process::os::OsProcessManager};
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let config = NetworkConfig::load_from_toml("./crates/examples/examples/0001-simple.toml")
         .expect("errored?");
 
-    let fs = LocalFileSystem;
-    let pm = OsProcessManager;
-    let provider = NativeProvider::new(fs.clone(), pm);
-    let orchestrator = Orchestrator::new(fs, provider);
-    orchestrator.spawn(config).await?;
+    Orchestrator::native().spawn(config).await?;
```

Comment:
How does this uses of the LocalFileSystem?

Reply:
I think Rust can infer the generics since it knows that native is implemented for T = LocalFileSystem